### ISSUE TITLE
can't use `XmirPackage` in package-info.java

### ISSUE THREAD

**Issue by yegor256** [objectionary/eo#3671]
At [this line](https://github.com/objectionary/eo/blob/master/eo-maven-plugin/src/main/java/org/eolang/maven/TranspileMojo.java#L337), in `TranspileMojo`, if we remove `//`, the build fails with internal Javac error (at least for me, with OpenJDK 23.0.1). Let's find out what's wrong and enable the usage of `XmirPackage`.

**Comment by yegor256** [comment]
@Suban05 I don't think this is the problem. It's perfectly legit to have multiple `package-info.java` files in the same package. We have that in many libraries. The bug is related to something else inside Javac.

**Comment by VasilevNStas** [comment]
@yegor256 this can be closed: the request is implemented and working.

## Evidence

1. The generated package-info now carries the annotation: `eo-maven-plugin/src/main/java/org/eolang/maven/PackageInfos.java:135` emits `@org.eolang.XmirPackage("...")`.
2. The transpiled `eo-runtime` package-info files contain it (e.g. `EO_range/package-info.java`, `EO_directory/package-info.java`), and `eo-runtime` compiles successfully with the annotation in place — the JDK 23.0.1 internal error from the report does not reproduce on the current code/JDK.
3. It is covered by tests: `PackageInfosTest` and `MjTranspileTest` assert the `@org.eolang.XmirPackage(...)` marker.
4. The referenced `TranspileMojo.java:337` no longer exists (the class was renamed/refactored).

The one open PR on this topic (#6204) was closed without merging, since the current code already produces the annotation through `PackageInfos`. Closing as done.
